test/load-parser-config.test.js

```diff
@@ -100,3 +102,12 @@ test('Throw error if "config" doesn`t exist', async (t) => {
 test('Throw error if "preset" doesn`t exist', async (t) => {
   await t.throwsAsync(loadParserConfig({ preset: "unknown-preset" }, { cwd }), { code: "MODULE_NOT_FOUND" });
 });
+
+test.serial("Load preset and config correctly when importFrom.silent fails", async (t) => {
+  sinon.stub(importFrom, "silent").returns(undefined);
+
+  await loadPreset(t, "angular");
+  await loadConfig(t, "angular");
```

Comment:
could we add assertions here to verify that the proper content is returned?

Reply:
I might be misunderstanding here, but `loadPreset` and `loadConfig` are custom ava helpers functions that check the loaded object contains a `headerPattern` property. Should I extend the ava helpers to actually load `conventional-changelog` and test the loaded content against that?

Reply:
you're right. i was definitely looking at it incorrectly. thanks for clarifying :)